src/Controller/EventsController.php

```diff
@@ -733,16 +734,25 @@ public function rejectAction(Request $request, PDO $db)
 
         $event_id     = $this->getItemId($request);
         $event_mapper = new EventMapper($db, $request);
+        $reason = filter_var($request->getParameter('reason'), FILTER_SANITIZE_STRING);
 
         if (!$event_mapper->thisUserCanApproveEvents()) {
             throw new Exception("You are not allowed to reject this event", Http::FORBIDDEN);
         }
 
-        $result = $event_mapper->reject($event_id, $request->user_id);
+        $result = $event_mapper->reject($event_id, $request->user_id, $reason);
```

Comment:
What do you think about wrapping the arguments to this method call?

Reply:
I typically will do this once the line hits 120 chars

Reply:
It could also be a `string` consisting of whitespace only.